src/Storage/Cookie.php

```diff
@@ -40,7 +55,7 @@ public function set($namespace, $key, $value)
         if (!headers_sent()) {
             // we do not throw an exception for now when headers already sent to not break the application
             // but could do later to make users aware there is an error
-            setcookie($name, $value, time() + (86400 * 365 * 2), $path = '/', "", false, $httpOnly = true);
+            setcookie($name, $value, time() + $this->cookieExpiry, $path = '/', "", false, $httpOnly = true);
```

Comment:
```suggestion
            // 0 = expire after session
            // negative value = force expiration
            $expireTime = $this->cookieExpiry == 0 ? 0 : time() + $this->cookieExpiry;
            setcookie($name, $value, $expireTime, $path = '/', "", false, $httpOnly = true);
```
We need this thing to be able to set expiration to 0 to indicate a session expire, rather than a specific timestamp always. We think its appropriate to set it to 0 when 0 is passed in as `cookieExpiry`